src/tex/sections/manually_sources_table.tex

```diff
@@ -34,5 +34,25 @@
        & Based on exercises for lectures in high-energy density materials at LMU Munich, Germany && \variable{output/question_count_per_dir/json_file_counts_hedm_munich.txt} \\
        & Based on pharmacology exams at the University of Vienna, Austria && \variable{output/question_count_per_dir/json_file_counts_pharmacology_vienna.txt} \\
        & Lab safety quizzes based on various sources && \variable{output/question_count_per_dir/json_file_counts_sci_lab_safety_test.txt} + \variable{output/question_count_per_dir/json_file_counts_lab_safety.txt} + \variable{output/question_count_per_dir/json_file_counts_stolaf.txt} + \variable{output/question_count_per_dir/json_file_counts_chemical_safety_mcq_exam.txt} + \variable{output/question_count_per_dir/json_file_counts_anderson.txt}\\
+\midrule
+Spectroscopy & Based on exams at the University of Hamburg, Germany && \variable{output/question_count_per_dir/json_file_counts_spectroscopy.txt} \\
+\midrule
+Solid state chemistry & Based on lectures published on the YouTube channels Materials Universe, Advanced Cyberinfrastructure Training at RPI, Phase Diagrams in Materials Science and Engineering" && 9\\
+                    & Based on the website \href{Wayback Machine}{https://archive.org/details/AshcroftSolidState/page/128/mode/2up} && 4\\
+                    & Based on the \href{https://ocw.mit.edu/courses/3-091sc-introduction-to-solid-state-chemistry-fall-2010/pages/organic-materials/29-polymers-synthesis-properties-applications/}{lecture notes} published by the Massachusetts Institute of Technology (MIT) && 4\\
+Technical chemistry & Based on notes from lectures at the University of Oviedo && 12\\
+                    & Based on the chemical engineering textbook from Octave Levenspiel && 8\\
+                    & Based on a lecture at the IIT Kharagpur, India && 1\\
+\midrule
+Hybridization questions &  Based on exams at the Ecole Polytechnique Federale de Lausanne && 9\\
+\midrule
+Biochemistry & Based on exams at the FSU Jena, Germany && 7\\
+\midrule
+Physical Chemistry & Based on a textbook && 6\\
```

Comment:
can we say what textbook?

Reply:
before we referred to it as "various textbooks"